daemon/graphdriver/windows/windows.go

```diff
@@ -412,7 +412,24 @@ func (d *Driver) Put(id string) error {
 }
 
 // Cleanup ensures the information the driver stores is properly removed.
+// We use this opportunity to cleanup any -removing folders which may be
+// still left if the daemon was killed while it was removing a layer.
 func (d *Driver) Cleanup() error {
+	// Note we don't return an error on cleanup failure, but do log a warning.
```

Comment:
This comment is a bit odd, because the first error below actually doesn't log anything 😅